personalities/ct-static-api/cert_checker.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2016 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ctfe
```

Comment:
I know this is a temporary home, but probably should either make the package  match the dir name, or just add a comment explaining why they're very different.

Reply:
Done, I've renamed it to sctfe until we find it a permanent home. sctfe could stand for SCT FE or static-certificate-transparency-fe hehe

Reply:
And `Starlight-CT-Frontend` too :)